### PR TITLE
amp-script: Fetch locally, add size limit

### PR DESCRIPTION
**Pull request by choumx**
Requires upgrade to include https://github.com/ampproject/worker-dom/pull/234.

- Upgrade `worker-dom` to 0.2.9.
- Fetch author script from AMP land so we can apply AMP CORS requirements. 
- Sets size limit to 150KB and enforce 1 amp-script element per page.

TODO: Refactor size limit to be global (shared across all amp-script elements) and remove 1 element per page constraint.

Uncompressed sizes of popular frameworks:

| Framework | KB |
| -- | -- |
| Preact | 8.2 |
| Vue | 85 |
| JQuery | 85 |
| React + ReactDOM | 110 |
| Angular | 180 |

### CODE REVIEW

**Comment by cramforce** [comment]
I think our last discussion arrived at a different limit. We can discuss later today.

File: extensions/amp-script/0.1/amp-script.js [hunk 1]
```diff
@@ -31,6 +32,8 @@ import {isExperimentOn} from '../../../src/experiments';
 /** @const {string} */
 const TAG = 'amp-script';
 
+/** @const {number} */
+const MAX_SCRIPT_SIZE = 150000;
```
Comment (jridgewell) [inline extensions/amp-script/0.1/amp-script.js]: Nit: add a comment

Comment (kristoferbaxter) [reply extensions/amp-script/0.1/amp-script.js]: Agreed, the table from the PR description helps readers to understand the limit as chosen.

File: package.json [hunk 2]
```diff
@@ -53,13 +53,14 @@
...
     "atob": "2.1.2",
-    "ava": "1.0.1",
     "autoprefixer": "9.4.3",
+    "ava": "1.0.1",
     "babel-eslint": "10.0.1",
     "babel-plugin-filter-imports": "2.0.4",
     "babel-plugin-istanbul": "5.1.0",
     "babel-plugin-transform-commonjs-es2015-modules": "4.0.1",
     "babel-plugin-transform-react-jsx": "6.24.1",
+    "babel-regenerator-runtime": "6.5.0",
```
Comment (kristoferbaxter) [inline package.json]: Intentional addition of `babel-regenerator-runtime`?

Comment (choumx) [reply package.json]: Must've been a bad merge. Removed, thanks.

**Pull request merged**